xonsh/aliases.py

```diff
@@ -65,9 +67,70 @@ def xexec(args, stdin=None):
         try:
             os.execvpe(args[0], args, denv)
         except FileNotFoundError as e:
-            return "xonsh: " + e.args[1] + ": " + args[0] + "\n"
+            return 'xonsh: ' + e.args[1] + ': ' + args[0] + '\n'
     else:
-        return "xonsh: exec: no args specified\n"
+        return 'xonsh: exec: no args specified\n'
+
+
+def history(args, stdin=None):
```

Comment:
Yeah, they were written by @aig787 originally and so they should just be considered stubs for the moment.  I imagine they'll come pretty quick once this is in :)